### DISCUSSION TITLE
Table with tab view

### DISCUSSION THREAD

**Discussion by xenocent** [orchidsoftware/platform#2146] [Q&A]
How do i config the table pagination with tab view, because when i send next page all table change to page 2 .
and how to turn off the pgaination in table ? sometimes i need to get only 1 page in table.
![image](https://user-images.githubusercontent.com/7766571/154848067-f227c278-4e79-499d-b472-a83a910f0452.png)
![image](https://user-images.githubusercontent.com/7766571/154848076-faa84fce-29ff-4197-b943-d453b7dca10f.png)

**Comment by ValerianMemsk** [comment]
Were you able to make any progress?
The problem appears to be in GET request method which uses default param name for pagination -> "page".
It could be solved by giving individual names to params tho i wasnt able to find HOW to do it.

**Reply by xenocent** [reply]
not yet i choose to make new page in the end, since no one answer this
